tox.ini

```diff
@@ -1,5 +1,31 @@
 [tox]
-envlist = copying,py27{,-color}
+envlist = copying,{py34,py35,py36,py37}-{lint,tests,color,cli}
+
+[testenv]
+whitelist_externals =
+    cli:   {[testenv:cli]whitelist_externals
+deps=
+    git+git://github.com/iot-lab/cli-tools.git#egg=iotlabcli
```

Comment:
Why not just `iotlab-cli` ?

Reply:
probably because `iotlabcli` on pypi is not the right version ?

Reply:
I don't know, it was already like this before.

Reply:
@aabadie what is the question ? I don't understand :)

Reply:
```
deps=
    iotlabcli
```
might be simpler, why use the github master version ?